torch/_prims/context.py

```diff
@@ -203,7 +184,17 @@ def _is_node_supported_nvfuser(node):
 
 def _is_func_unsupported_nvfuser(torch_function_mode, func, args, kwargs):
     with torch_function_mode:
-        gm = get_isolated_graphmodule(func, args, kwargs)
+        try:
+            gm = get_isolated_graphmodule(func, args, kwargs)
+        except Exception as e:
```

Comment:
I'm not a big fan of these `catch it all` during early prototyping phase. But at least it's throwing a warning, so hopefully we'll keep an eye out for these.

Reply:
🤞